examples/rest/merchant/submit_invoice.py

```diff
@@ -0,0 +1,43 @@
+# python -c "import examples.rest.merchant.submit_invoice"
```

Comment:
can we add later support for the rest of merchant endpoints please? it's really useful for 3rd parties
https://docs.bitfinex.com/reference/merchant-endpoints